### PR TITLE
Added optional tagline - default not showing

### PR DESCRIPTION
**Pull request by pederan**

### CODE REVIEW

File: assets/template.php [hunk 1]
```diff
@@ -48,6 +48,15 @@
 				);
 			}
 
+			if ( ! empty( $this->tagline ) ) {
+				printf(
+					'<div class="tagline">%s</div>',
```
Comment (walbo) [inline assets/template.php]: Prefix the classname with `hogan-`

File: class-banner.php [hunk 2]
```diff
@@ -218,6 +218,18 @@ public function get_fields(): array {
...
 
+			/*
+			 * Tagline field
+			 */
+			if ( true === apply_filters( 'hogan/module/banner/tagline/enabled', false ) ) {
+				$content_tab[] = apply_filters( 'hogan/module/banner/acf/tagline', [
+					'type'  => 'text',
+					'key'   => $this->field_key . '_tagline',
+					'name'  => 'tagline',
+					'label' => __( 'Add tagline', 'hogan-banner' ),
```
Comment (walbo) [inline class-banner.php]: Think we can drop the "Add" and just have "Tagline"

File: class-banner.php [hunk 3]
```diff
@@ -506,6 +521,11 @@ public function load_args_from_layout_content( array $raw_content, int $counter
 				$this->image = null;
 			}
 
+			// Tagline.
+			if ( true === apply_filters( 'hogan/module/banner/tagline/enabled', false ) ) {
+				$this->tagline = $raw_content['tagline'];
```
Comment (walbo) [inline class-banner.php]: Should define the $tagline var in top of this class.

Comment (pederan) [reply class-banner.php]: Label is a property that is used on the module instance. Can we keep this as tagline, that is what is used already for DSS?

**Pull request merged**